cmd/lncli/commands.go

```diff
@@ -1972,7 +1972,7 @@ var updateChannelPolicyCommand = cli.Command{
 	broadcast to the rest of the network within the next batch.
 	Channel points are encoded as: funding_txid:output_index`,
 	Flags: []cli.Flag{
-		cli.Int64Flag{
+		cli.StringFlag{
```

Comment:
I don't think switching all the types of these values (which are clearly integers) is the best way to fix the help text here. Couldn't we instead just explicitly set the default values, or indicate that it doesn't have one?

Reply:
Setting the default value is tricky too. Because then if you forget to set the param to a custom value it will reset your channel to the defaults. Unless you explicitly check with `ctx.IsSet()`, I guess. But then it might still look to the user like it would be overwritten with the default value if not specified...

Reply:
We can [set default values for help output](https://cli.urfave.org/v2/#default-values-for-help-output), if we are willing to go to `v2`. Meanwhile I don't think we should make everything a string to avoid the `default:0` part. I think even with `v1` you can [generate customized help text?](https://cli.urfave.org/v1/#generated-help-text).

Reply:
I added an alternative PR using v2 of the CLI library (#6778)

Reply:
These comments all make sense to me, but I am unsure how to proceed. @C-Otto does your PR make this one obselete? 

In my experience, the `updatechanpolicy` is used frequently to set some but not all values, with base & fee rates being the most popular. For this reason, I don't think any defaults / unset flags should be used to change policy.

Reply:
With v2 you don't have to use strings all over the place. However, for integer values it's not possible to get rid of the "Default: " entry completely. I don't know if my approach is strictly better.